hipBLASLt does expect ro find all msgpack implementations via the name msgpack.
This will not work with the actual packages we use in EasyBuild.

Author: Jan Andre Reuter (jan@zyten.de)

diff --color -Naur hipBLASLt-rocm-6.4.1.orig/tensilelite/Tensile/Source/lib/CMakeLists.txt hipBLASLt-rocm-6.4.1/tensilelite/Tensile/Source/lib/CMakeLists.txt
--- hipBLASLt-rocm-6.4.1.orig/tensilelite/Tensile/Source/lib/CMakeLists.txt     2025-04-25 19:53:57.000000000 +0200
+++ hipBLASLt-rocm-6.4.1/tensilelite/Tensile/Source/lib/CMakeLists.txt  2025-06-16 13:24:55.602414423 +0200
@@ -96,7 +96,7 @@
 endif()

 if(TENSILE_USE_MSGPACK)
-    find_package(msgpack REQUIRED)
+    find_package(msgpack-cxx REQUIRED)
     target_compile_definitions(TensileHost PUBLIC -DTENSILE_MSGPACK=1)

     if(TARGET msgpackc-cxx)
